tslint.json

```diff
@@ -21,6 +24,9 @@
     ],
     "no-default-export": true,
     "no-floating-promises": false,
-    "no-console": true
+    "no-console": true,
+    "no-unused-expression": false,
```

Comment:
`no-unused-expression` was causing issue with some chai assertions such as `empty`, turning it off and using a setting from an npm package `no-unused-expression-chai` enables all of the checking from `no-unused-expression` without causing linting errors with these assertions.

Reply:
Very nice but do we need to disable it here? Doc does not mention it and for me it looks like new module disables `no-unused-expression` out of the box.

Reply:
The `tslint-config-standard` we extend our config from explicitly sets `no-unused-expression` to true, requiring us to turn it off here or it causes linting errors.